### PR TITLE
Better transitions (chapters 5-8)

### PR DESCRIPTION
**Pull request by johko**
Another batch of slight format adaptions and some introductions.

Mainly covering chapters 5-8.

### CODE REVIEW

File: chapters/en/Unit 7 - Video and Video Processing/introduction-to-video.mdx [hunk 1]
```diff
@@ -1,7 +1,17 @@
 # Introduction
+
+Welcome to the Video and Video Processing unit. Maybe you have realized that in our course content so far we mainly focused on standard, static 2D images.
```
Comment (SuryaKrishna02) [inline chapters/en/Unit 7 - Video and Video Processing/introduction-to-video.mdx:3]:
```suggestion
Welcome to the Video and Video Processing unit. Maybe you have realized that in our course content so far, we have mainly focused on standard, static 2D images.
```

File: chapters/en/Unit 7 - Video and Video Processing/introduction-to-video.mdx [hunk 2]
```diff
@@ -1,7 +1,17 @@
 # Introduction
+
+Welcome to the Video and Video Processing unit. Maybe you have realized that in our course content so far we mainly focused on standard, static 2D images.
+Of course the real world of Computer Vision has a lot more to offer. Videos are definitely one of the most used mediums in our world due to applications
```
Comment (SuryaKrishna02) [inline chapters/en/Unit 7 - Video and Video Processing/introduction-to-video.mdx:4]:
```suggestion
Of course, the real world of Computer Vision has a lot more to offer. Videos are definitely one of the most used mediums in our world due to applications like Social Media, broadcasts, or surveillance cameras.
```

File: chapters/en/Unit 7 - Video and Video Processing/introduction-to-video.mdx [hunk 3]
```diff
@@ -1,7 +1,17 @@
 # Introduction
+
+Welcome to the Video and Video Processing unit. Maybe you have realized that in our course content so far we mainly focused on standard, static 2D images.
+Of course the real world of Computer Vision has a lot more to offer. Videos are definitely one of the most used mediums in our world due to applications
+like Social Media, broadcasts or surveillance cameras.
```
Comment (SuryaKrishna02) [inline chapters/en/Unit 7 - Video and Video Processing/introduction-to-video.mdx:5]:
```suggestion
```

File: chapters/en/Unit 7 - Video and Video Processing/introduction-to-video.mdx [hunk 4]
```diff
@@ -1,7 +1,17 @@
 # Introduction
+
+Welcome to the Video and Video Processing unit. Maybe you have realized that in our course content so far we mainly focused on standard, static 2D images.
+Of course the real world of Computer Vision has a lot more to offer. Videos are definitely one of the most used mediums in our world due to applications
+like Social Media, broadcasts or surveillance cameras.
+
+Given their importance in our society and research, we also want to talk about them here in our course. In this introduction chapter you will learn some very basic theory
```
Comment (SuryaKrishna02) [inline chapters/en/Unit 7 - Video and Video Processing/introduction-to-video.mdx:7]:
```suggestion
Given their importance in our society and research, we also want to talk about them here in our course. In this introduction chapter, you will learn some very basic theory behind videos before going on to have a closer look at video processing.
```

File: chapters/en/Unit 7 - Video and Video Processing/introduction-to-video.mdx [hunk 5]
```diff
@@ -1,7 +1,17 @@
 # Introduction
+
+Welcome to the Video and Video Processing unit. Maybe you have realized that in our course content so far we mainly focused on standard, static 2D images.
+Of course the real world of Computer Vision has a lot more to offer. Videos are definitely one of the most used mediums in our world due to applications
+like Social Media, broadcasts or surveillance cameras.
+
+Given their importance in our society and research, we also want to talk about them here in our course. In this introduction chapter you will learn some very basic theory
+behind videos before going on to have a closer look at video processing.
```
Comment (SuryaKrishna02) [inline chapters/en/Unit 7 - Video and Video Processing/introduction-to-video.mdx:8]:
```suggestion
```

File: chapters/en/Unit 8 - 3D Vision, Scene Rendering and Reconstruction/Introduction/introduction.mdx [hunk 6]
```diff
@@ -0,0 +1,21 @@
+# Introduction
+
+Apart from videos, about which we talked in the last chapter, another common form of visual data comes in the 3-dimensional form.
+While for 2D images we usually have the two dimensions, commonly labelled as _x_ and _y_, for 3D images we have three dimensions, referred to as _x_, _y_ and _z_.
+
+"But wait", I hear you say, "videos also have three dimensions!" That is completely correct - videos have the two spatial dimensions _x_ and _y_ and the
```
Comment (SuryaKrishna02) [inline chapters/en/Unit 8 - 3D Vision, Scene Rendering and Reconstruction/Introduction/introduction.mdx:6]:
```suggestion
"But wait," I hear you say, "videos also have three dimensions!" That is completely correct - videos have the two spatial dimensions, _x_ and _y_, and the temporal dimension, _t_. The difference with 3D data is that here all three dimensions are of a spatial nature. This helps us to create a better model of our world and our perceptive capabilities. That is why one very common field for 3D applications nowadays is Mixed Reality applications, in which we try to merge the digital and analog worlds.
```

File: chapters/en/Unit 8 - 3D Vision, Scene Rendering and Reconstruction/Introduction/introduction.mdx [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+temporal dimension _t_. The difference to 3D data is, that here all three dimensions are of a spatial nature. This helps us to create a better model of our
```
Comment (SuryaKrishna02) [inline chapters/en/Unit 8 - 3D Vision, Scene Rendering and Reconstruction/Introduction/introduction.mdx:7]:
```suggestion
```

File: chapters/en/Unit 8 - 3D Vision, Scene Rendering and Reconstruction/Introduction/introduction.mdx [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+world and perceptive capabilities. That is why one very common field for 3D applications nowadays are Mixed Reality applications, in which we try to merge the digital and
```
Comment (SuryaKrishna02) [inline chapters/en/Unit 8 - 3D Vision, Scene Rendering and Reconstruction/Introduction/introduction.mdx:8]:
```suggestion
```

File: chapters/en/Unit 8 - 3D Vision, Scene Rendering and Reconstruction/Introduction/introduction.mdx [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+the analog world.
```
Comment (SuryaKrishna02) [inline chapters/en/Unit 8 - 3D Vision, Scene Rendering and Reconstruction/Introduction/introduction.mdx:9]:
```suggestion
```

File: chapters/en/Unit 8 - 3D Vision, Scene Rendering and Reconstruction/Introduction/introduction.mdx [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+
+## Unit Overview
+
+You will learn more about applications of 3D Computer Vision in the first chapter after this introduction. Right after that we will take a look at the historical
```
Comment (SuryaKrishna02) [inline chapters/en/Unit 8 - 3D Vision, Scene Rendering and Reconstruction/Introduction/introduction.mdx:13]:
```suggestion
You will learn more about applications of 3D Computer Vision in the first chapter after this introduction. Right after that, we will take a look at the historical developments of 3D applications - all the way from the 19th century to today.
```

File: chapters/en/Unit 8 - 3D Vision, Scene Rendering and Reconstruction/Introduction/introduction.mdx [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+developments of 3D applications - all the way from the 19th century to today.
```
Comment (SuryaKrishna02) [inline chapters/en/Unit 8 - 3D Vision, Scene Rendering and Reconstruction/Introduction/introduction.mdx:14]:
```suggestion
```

File: chapters/en/Unit 8 - 3D Vision, Scene Rendering and Reconstruction/Introduction/introduction.mdx [hunk 12]
```diff
@@ -0,0 +1,21 @@
...
+
+After these general topics, we'll dive right into the terminologies and concepts with three chapters about camera models, linear algebra and different representations.
+
+We are following up the theory with some proper fields of use for 3D Computer Vision. Starting of with Novel View Synthesis, followed by Stereo Vision and finishing
```
Comment (SuryaKrishna02) [inline chapters/en/Unit 8 - 3D Vision, Scene Rendering and Reconstruction/Introduction/introduction.mdx:18]:
```suggestion
We are following up the theory with some proper fields of use for 3D Computer Vision. Starting off with Novel View Synthesis, followed by Stereo Vision and finishing this unit (for now) with one of the most popular applications right now - Neural Radiance Fields (NeRFs).
```

File: chapters/en/Unit 8 - 3D Vision, Scene Rendering and Reconstruction/Introduction/introduction.mdx [hunk 13]
```diff
@@ -0,0 +1,21 @@
...
+this unit (for now) with one of the most popular applications right now - Neural Radiance Fields (NeRFs).
```
Comment (SuryaKrishna02) [inline chapters/en/Unit 8 - 3D Vision, Scene Rendering and Reconstruction/Introduction/introduction.mdx:19]:
```suggestion
```

**Review by SuryaKrishna02 — APPROVED** [review]
Looks Great. Left some minor formatting changes.

**Comment by johko** [comment]
Thank you @SuryaKrishna02 :pray:  You might have noticed I'm not very great with commas :sweat_smile:

**Review by SuryaKrishna02 — APPROVED** [review]
Looks Great!😀

**Review by merveenoyan — APPROVED** [review]

**Comment by merveenoyan** [comment]
@johko can we find another reviewer to merge this? maybe @SuryaKrishna02 ?

**Review by SuryaKrishna02 — APPROVED** [review]
@merveenoyan @johko Weirdly my approval of changes is not considered as approval. But few days back it was showing as approval. 🤔

**Comment by merveenoyan** [comment]
@SuryaKrishna02 no worries!
@lunarflu can you approve?

**Pull request merged**